gen/definitions/internal_user.yaml

```diff
@@ -62,27 +63,38 @@ attributes:
     type: String
     description: Last name of the internal user
     example: Doe
+  - model_name: identityGroups
+    data_path: [InternalUser]
+    type: String
+    description: CSV of identity group IDs.
+    example: "d68f6180-66a2-11ee-9cc1-9eb2a3ecc82a"
+    exclude_test: true
+  - model_name: customAttributes
+    data_path: [InternalUser]
+    type: String
+    description: Key value map
+    exclude_test: true
   - model_name: passwordIDStore
     data_path: [InternalUser]
     tf_name: password_id_store
     type: String
     description: The ID store where the internal user's password is kept
     default_value: Internal Users
     example: Internal Users
-  # - model_name: expiryDateEnabled
-  #   data_path: [InternalUser]
-  #   tf_name: expiry_date_enabled
-  #   type: Bool
-  #   description: Enable a password expiry date
-  #   default_value: false
-  #   example: false
-  # - model_name: expiryDate
-  #   data_path: [InternalUser]
-  #   tf_name: expiry_date
-  #   type: String
-  #   description: Password expiry date
-  #   example: "2024-12-11"
-  #   exclude_test: true
+  - model_name: expiryDateEnabled
```

Comment:
I did not get this to work. Have you tested the use of these attributes?

Reply:
yes i tested those attributes and they work